zooniverse/front-end-monorepo
# lib-classifier: Refactor survey task carousel control arrows and 3 column styling

## Package
- lib-classifier

## Linked Issue and/or Talk Post
- closes #4557 
- addresses ChoiceButton text overflow per [this comment](https://github.com/zooniverse/front-end-monorepo/pull/4841#pullrequestreview-1488373259)

## Describe your changes
- refactors survey task carousel arrows to size `large` and color `neutral-2` (gold)
- styles ChoiceButton to flex column when Choices are 3 columns
- ^ I considered adding `{ word-break: break-word; }` to the ChoiceButton StyledBox `span` styling, but after testing at multiple screen widths feel it's preferable as is

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX?
  - lib-classifier staging survey task workflow - https://local.zooniverse.org:8080/?project=1634&workflow=3729
  - app-project staging survey task workflow - https://local.zooniverse.org:3000/projects/markb-panoptes/survey-task-test-project/classify/workflow/3729
- What user actions should my reviewer step through to review this PR?
  - click a ChoiceButton, note carousel control arrow styling
  - click a ChoiceButton, click a Confusion, note carousel control arrow styling
  - (optionally zoomed in a little) check [survey task at 3 columns with thumbnails](https://local.zooniverse.org:3000/projects/markb-panoptes/survey-task-test-project/classify/workflow/3729) and adjust screen width, noting hopefully no choice button text overflow
- Which storybook stories should be reviewed?
  - http://localhost:6006/?path=/story/tasks-survey--default
- Are there plans for follow up PR’s to further fix this bug or develop this feature? no

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## Refactoring
